acceptance/tests/rototillerTask/command_arguments_with_override.rb

```diff
@@ -0,0 +1,78 @@
+require 'beaker/hosts'
+require 'rakefile_tools'
+
+test_name 'C97827: can set envvar to override command name when using task.command' do
+  extend Beaker::Hosts
+  extend RakefileTools
+
+  step 'Test command with an override_env that has a value' do
+
+    override_env = 'BROCKSAMSON'
+    argument_override_env = 'RUSTYVENTURE'
+    env_key = 'THIS_WAS_IN_ENV'
+    argument_env_key = 'THIS_WAS_THE_ARG_IN_THE_ENV'
+    env_value = 'echo ' << env_key
+
+    default_arg = 'HANKVENTURE'
+
+    @task_name    = 'command_with_args_and_defaults'
+    rakefile_contents = <<-EOS
+$LOAD_PATH.unshift('/root/rototiller/lib')
+require 'rototiller'
+
+Rototiller::Task::RototillerTask.define_task :#{@task_name} do |t|
+    t.add_command({:name => 'echo', :override_env => '#{override_env}', :argument => '#{default_arg}', :argument_override_env => '#{argument_override_env}'})
+end
+    EOS
+    rakefile_path = create_rakefile_on(sut, rakefile_contents)
```

Comment:
we're starting to repeat a bunch of this stuff in tests.
perhaps create_rakefile_on should take a bunch of task block strings
the rest of this crap is implied (first three lines).
can do in another ticket.  probably should have a Rakefile class that has a bunch of task classes composed into it.
</thinking outloud>